src/app/shared/pipes/difference-in-time.pipe.ts

```diff
@@ -0,0 +1,36 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'differenceInTime'
+})
+export class DifferenceInTimePipe implements PipeTransform {
+
+//   transform(value: Date): string {
+//     const difference = Math.abs(new Date().getTime() - new Date(value).getTime());
+//     const diffInMinutes = difference / 1000 / 60 / 60;
+    
+//     if (diffInMinutes >= 60) {
+//       return `${Math.round(diffInMinutes / 60)} hours ago`;
+//     } 
+//       return `${diffInMinutes} mins ago`;
+//   }
+
+transform(value: Date): string {
+  const difference = Math.abs(new Date().getTime() - new Date(value).getTime());
+  const diffInMinutes = difference / 1000 / 60;
+  console.log(diffInMinutes);
```

Comment:
console.log and debuggers are unacceptable to be present in PR. please always check your code and do not keep console.log and debuggers in your app.

Reply:
fixed